byron/ledger/executable-spec/cs-ledger.cabal

```diff
@@ -109,3 +109,36 @@ test-suite ledger-delegation-test
                "-with-rtsopts=-K1m -M50m"
   if (!flag(development))
     ghc-options: -Werror
+
+test-suite ledger-sts-test
+  hs-source-dirs: test
+  main-is: STS.hs
+  exposed-modules: Generator
+                 , Ledger.Pvbump.Properties
+  other-modules:   Generator
```

Comment:
These should only be in one section, exposed or other

Reply:
Expose everything, particularly in testing libraries.